src/cli/README.md

```diff
@@ -3728,6 +3729,32 @@ Reset the TX queue time-in-queue statistics.
 Done
 ```
 
+### tmfhostcheck enable
```

Comment:
To be more concise, maybe we can merge them into a single heading like https://github.com/openthread/openthread/blob/main/src/cli/README.md#dns-compression-enabledisable